src/Model/Entity/Asset.php

```diff
@@ -282,12 +282,12 @@ public function moveToWebroot(?string $filename = null, ?string $path = 'files',
      *  If set to true, new Versions will not be detected. You will have to change the filename accordingly, or empty the folder set in $path.
      *
      * @param string|null $filename Name of the file. Default will be the Asset's id (uuid)
-     * @param string|null $path Path from webroot where the copy will be available
+     * @param string $path Path from webroot where the copy will be available
      * @param array $config See above
      * @return string
      * @throws \Assets\Error\FileNotFoundException
      */
-    public function copyToWebroot(?string $filename = null, ?string $path = 'files', array $config = []): string
```

Comment:
You don't check if `$path` is null inside your `copyToWebroot()` but simply expect it to be a valid string. Since we basically require it to be a valid string path it should not be nullable according to the param type.